src/Sentry.EntityFramework/Sentry.EntityFramework.csproj

```diff
@@ -11,7 +11,7 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net461'">
-    <PackageReference Include="EntityFramework" Version="6.0.0" />
+    <PackageReference Include="EntityFramework" Version="6.2.0" />
```

Comment:
Add a note to the migration guide? That from version Xyz we'll require EF 6.2

Reply:
https://github.com/getsentry/sentry-docs/compare/master...SimonCropp:patch-2